front/package.json

```diff
@@ -53,8 +53,6 @@
     "d3": "^7.8.5",
     "d3-brush": "^3.0.0",
     "d3-drag": "^3.0.0",
```

Comment:
Are these 2 still used ?

Reply:
No you're right ! I removed them, thanks :)